.github/workflows/docker.yml

```diff
@@ -0,0 +1,34 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths-ignore:
+      - 'config.example.toml'
+      - '.gitignore'
+      - 'docker-compose.yml'
```

Comment:
Why is the compose and example config file excluded?

Reply:
I'll prefer it if you can remove those two files from being excluded.

Changes to the compose (which will probably be edited to use the image) and the example config is something I'll like to keep updated in the image